composer.json

```
@@ -124,6 +124,11 @@
             "drush user:password editor editor",
             "drush user:unblock editor"
         ],
+        "drupal:rebuild": [
+            "sudo rm -rf vendor web composer.lock | true",
+            "composer install",
```

```suggestion
            "@composer install",
```
To re-invoke the same Composer instance that is already running this script.